index.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-var dotProp = require('dot-prop');
+var dotPropGet = require('dot-prop').get;
```

Comment:
Fine. Is it your coding style not to cache the used function directly? I thought it was slightly more performant?